README.md

```diff
@@ -203,3 +203,432 @@ kubectl delete -k subscription/deployment-k8s-regional-rbd
    ```
 
    At this point the application is managed again by *OCM*.
+
+## Deploy OCM discovered application
+
+The sample application is configured to run on cluster `dr1`. To deploy
+it on cluster `dr1` and make it possible to fail over or relocate to
+cluster `dr2` we need to create the namespace on both clusters:
+
+```
+kubectl create ns deployment-rbd --context dr1
+kubectl create ns deployment-rbd --context dr2
+```
+
+To deploy the application apply the deployment-rbd workload to the
+`deployment-rbd` namespace on cluster `dr1`:
+
+```
+kubectl apply -k workloads/deployment/k8s-regional-rbd -n deployment-rbd --context dr1
+```
+
+To view the deployed application use:
+
+```
+kubectl get deploy,pod,pvc -n deployment-rbd --context dr1
+```
+
+Example output:
+
+```
+NAME                      READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/busybox   1/1     1            1           24s
+
+NAME                           READY   STATUS    RESTARTS   AGE
+pod/busybox-6bbf88b9f8-fz2kn   1/1     Running   0          24s
+
+NAME                                STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      VOLUMEATTRIBUTESCLASS   AGE
+persistentvolumeclaim/busybox-pvc   Bound    pvc-c45a3892-167b-4dbc-a250-09c5f288c766   1Gi        RWO            rook-ceph-block   <unset>                 24s
+```
+
+## Enabling DR for OCM discovered application
+
+Unlike OCM managed applications, the DR resources for all applications
+are in the `ramen-ops` namespace.
+
+To prepare the `ramen-ops` namespaces apply the managed clusterset
+binding resource. This should be done once before enabling DR for
+discovered applications.
+
+```
+kubectl apply -f dr/discovered/ramen-ops/binding.yaml --context hub
+```
+
+Example output:
+
+```
+managedclustersetbinding.cluster.open-cluster-management.io/default created
+```
+
+To enable DR for the application, apply the DR resources to the hub
+cluster:
+
+```
+kubectl apply -k dr/discovered/deployment-rbd --context hub
+```
+
+Example output:
+
+```
+placement.cluster.open-cluster-management.io/deployment-rbd-placement created
+placementdecision.cluster.open-cluster-management.io/deployment-rbd-placement-decision created
+drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc created
+```
+
+To set the application placement, patch the placement decision resource
+in the `ramern-ops` namespace on the hub:
+
+```
+kubectl patch placementdecision deployment-rbd-placement-decision \
+    --subresource status \
+    --patch '{"status": {"decisions": [{"clusterName": "dr1", "reason": "dr1"}]}}' \
+    --type merge \
+    --namespace ramen-ops \
+    --context hub
+```
```

Comment:
This step should not be needed if we remove the placemnt-decision.yaml - ramen will create a new configured resource automatically.
https://github.com/RamenDR/ramen/blob/d475bc759f4cb84b0fd823b9eea2b0b54b7c0364/controllers/drplacementcontrol_controller.go#L2116